jwst/assign_wcs/tests/test_nirspec.py

```diff
@@ -57,7 +57,7 @@ def create_hdul():
     phdu.header['instrume'] = 'NIRSPEC'
     phdu.header['detector'] = 'NRS1'
     phdu.header['time-obs'] = '8:59:37'
-    phdu.header['date-obs'] = '2014-09-05'
+    phdu.header['date-obs'] = '2016-09-05'
```

Comment:
Yes, that's the hope!  Right now the `assign_wcs` code these tests are testing only works on CV3 files.  Therefore we need to make these tests use CV3 reffiles, and thus have an appropriate USEAFTER date so CRDS selects them.